molo/globalsite/templatetags/globalsite_tags.py

```diff
@@ -0,0 +1,11 @@
+from django import template
+from wagtail.wagtailcore.models import Site
+
+register = template.Library()
+
+
+@register.simple_tag(takes_context=True)
+def current_country(context):
```

Comment:
This doesn't appear to be used. Also, you should have access to `request.site` by default.